llarp/context.cpp

```diff
@@ -166,6 +166,14 @@ namespace llarp
       SigINT();
     }
 #ifndef _WIN32
+    if (sig == SIGUSR1)
+    {
+      if (router and not router->IsServiceNode())
```

Comment:
Why do we not want this to be possible for a service node?

Reply:
it is a state that a service node should never get into and should be a hard failure if it is. allowing a service node to do this provides no good things only bad things, if a service node can kill all of its outbound link sessions on command it feels like only bad things can happen from allowing admins to do that.